pod/perlpacktut.pod

```diff
@@ -42,9 +42,9 @@ of these two functions.
 To see how (un)packing works, we'll start with a simple template
 code where the conversion is in low gear: between the contents of a byte
 sequence and a string of hexadecimal digits. Let's use C<unpack>, since
-this is likely to remind you of a dump program, or some desperate last
-message unfortunate programs are wont to throw at you before they expire
-into the wild blue yonder. Assuming that the variable C<$mem> holds a 
+this is likely to remind you of a dump program, or some terse panic
```

Comment:
While I think your phrasing is an improvement, I have to wonder why `unpack` is likely to remind me of a dump program or a panic message.

Reply:
This could be improved by swapping the order of the sentence, like "Since this is likely to remind you..., let's use C<unpack>."